src/components/m-dropdown/index.scss

```diff
@@ -80,6 +84,13 @@
 
 .m-dropdown__icon {
   @include dropdown-icon();
+  pointer-events: none;
```

Comment:
@LucaMele 
@jackblackCH 
If any style changes are required, they should be either added to an existing mixin or provided by a nother mixin

Reply:
Agree, this is not a style change though. `pointer-events` is added on purpose in this situation (to allow clicks through that icon)

Reply:
Cool.
I would argue that it makes sense to have `pointer-events:none` in general for drop down icons

Reply:
neutral here on my side

Reply:
for dropdown icons yes, I don't see a benefit at all to put things into a mixin if you don't reuse it. In this case, if you need the dropdown icon clickable, you are stuck.